### PR TITLE
WT-10492 Create a new python test to start testing both providers

### PR DESCRIPTION
**Pull request by lting0804**
- Added Azure Store and GCP Store as a storage source in `test_tiered19.py` and called make scenarios

### CODE REVIEW

File: test/suite/helper_tiered.py [hunk 1]
```diff
@@ -145,13 +149,33 @@ def gen_tiered_storage_sources(random_prefix='', test_name='', tiered_only=False
             bucket_prefix2 = generate_s3_prefix(random_prefix, test_name),
             num_ops=20,
             ss_name = 's3_store')),
+        ('azure_store', dict(is_tiered = True,
```
Comment (jiechenbo) [inline test/suite/helper_tiered.py:152]: Remove this please, since we aren't using it.

File: test/suite/helper_tiered.py [hunk 2]
```diff
@@ -145,13 +149,33 @@ def gen_tiered_storage_sources(random_prefix='', test_name='', tiered_only=False
...
+            num_ops=100,
+            ss_name = 'gcp_store')),
         ('non_tiered', dict(is_tiered = False)),
     ]
 
     # Return a sublist to use for the tiered test scenarios as last item on list is not a scenario
     # for the tiered tests.  
     if tiered_only:
-        return tiered_storage_sources[:2]
+        return tiered_storage_sources[:(len(tiered_storage_sources)-1)]
```
Comment (jiechenbo) [inline test/suite/helper_tiered.py:178]: ditto.

File: test/suite/helper_tiered.py [hunk 3]
```diff
@@ -59,7 +63,7 @@ def get_auth_token(storage_source):
 def get_bucket_name(storage_source, i):
     if storage_source == 's3_store':
         return s3_buckets[i]
-    if storage_source == 'dir_store':
+    else:
```
Comment (jiechenbo) [inline test/suite/helper_tiered.py:66]: revert this please.

File: test/suite/test_tiered18.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+FileSystem = wiredtiger.FileSystem  # easy access to constants
+
+class test_tiered06(wttest.WiredTigerTestCase, TieredConfigMixin):
+
+    tiered_storage_sources = [
+        ('azure_store', dict(is_tiered = True,
+            is_local_storage = False,
+            auth_token = get_auth_token('azure_store'), 
+            bucket = get_bucket_name('azure_store', 0),
```
Comment (jiechenbo) [inline test/suite/test_tiered18.py:15]: Can we place the bucket names we are currently using in our personal accounts here? Or create a new bucket name uses primary for testing?

File: test/suite/helper_tiered.py (hunk 1 above)
Comment (lting0804) [reply test/suite/helper_tiered.py]: addressed https://github.com/wiredtiger/wiredtiger/pull/8707/commits/61b13a704e7d58933ead288cc550b6b4492ad90c

File: test/suite/helper_tiered.py (hunk 2 above)
Comment (lting0804) [reply test/suite/helper_tiered.py]: addressed https://github.com/wiredtiger/wiredtiger/pull/8707/commits/61b13a704e7d58933ead288cc550b6b4492ad90c

File: test/suite/helper_tiered.py (hunk 3 above)
Comment (lting0804) [reply test/suite/helper_tiered.py]: addressed https://github.com/wiredtiger/wiredtiger/pull/8707/commits/61b13a704e7d58933ead288cc550b6b4492ad90c

File: test/suite/test_tiered18.py (hunk 4 above)
Comment (lting0804) [reply test/suite/test_tiered18.py]: addressed in https://github.com/wiredtiger/wiredtiger/pull/8707/commits/2293e04b33409f7e91539c11a17b4643ccb29140

**Review by jiechenbo — APPROVED** [review]
LGTM!

**Review by sulabhM — APPROVED** [review]

**Comment by lting0804** [comment]
evergreen merge

**Comment by lting0804** [comment]
evergreen merge

**Pull request merged**